UIKitDZ/UIKitDZ/Controllers/LoginViewController.swift

```diff
@@ -0,0 +1,218 @@
+//
+//  LoginViewController.swift
+//  UIKitDZ
+//
+//  Created by 2lup on 03.10.2022.
+//
+
+import UIKit
+
+// MARK: - LoginViewController
+
+/// Экран авторизации
+final class LoginViewController: UIViewController {
+    
+    // MARK: - IBOutlet
+    
+    @IBOutlet private weak var nicknameTextView: UITextField!
+    @IBOutlet private weak var emailTextView: UITextField!
+    @IBOutlet private weak var passwordTextView: UITextField!
+    @IBOutlet private weak var registrationButton: UIButton!
+    @IBOutlet private weak var enterButton: UIButton!
+    
+    // MARK: - Private Properties
+    
+    private var usersArray: [UserData] = []
```

Comment:
```suggestion
    private var users: [UserData] = []
```
по букве s понятно что это массив